src/swagger/get.ts

```diff
@@ -14,23 +14,6 @@ enum SwaggerDefaultConfig {
 export const updateSwagger = (swagger: ISwaggerDefinition, key: string, values: object) =>
   Object.assign(swagger[key], values);
 
-const getPath = (path: string, prefix: string, shouldOmitPrefix: boolean) => {
-  const cleanedPath = cleanPath(path);
-
-  if (shouldOmitPrefix) {
-    const prefixRegex = new RegExp(`^${prefix}`);
-    const newPath = cleanedPath.replace(prefixRegex, '');
-
-    if (newPath === '') {
-      return '/';
-    }
-
-    return newPath;
-  }
-
-  return cleanedPath;
-};
-
```

Comment:
Similar logic moved to the `getTransformedPath, makeRemovePrefix` imports above.